Source/Extensions/Blazorise.Captcha/Captcha.razor.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Captcha;
+
+public partial class Captcha : BaseComponent
+{
+    /// <summary>
+    /// The current state of the Captcha.
+    /// </summary>
+    public CaptchaState State { get; private set; } = new();
+
+    /// <summary>
+    /// A Captcha solved event. 
+    /// Provides contextual information about the Captcha state after the user has solved.
+    /// </summary>
+    [Parameter] public EventCallback<CaptchaState> OnSolved { get; set; }
+
+    /// <summary>
+    /// A Captcha validation event. Further validation may be performed here.
+    /// </summary>
+    [Parameter] public Func<CaptchaState, Task<bool>> OnValidate { get; set; }
+
+    /// <summary>
+    /// The Captcha expired event.
+    /// </summary>
+    [Parameter] public EventCallback OnExpired { get; set; }
+
+
+    protected override async Task OnAfterRenderAsync( bool firstRender )
+    {
+        if ( firstRender )
+        {
+            await Render();
```

Comment:
I think this should be called `InitializeCaptcha` or similar. Not `Render`.

Reply:
Well, it does renders the captcha in practice... hehe